tools/coqdep.ml

```diff
@@ -80,6 +80,8 @@ let rec parse = function
   | "-dyndep" :: "both" :: ll -> option_dynlink := Both; parse ll
   | "-dyndep" :: "var" :: ll -> option_dynlink := Variable; parse ll
   | ("-h"|"--help"|"-help") :: _ -> usage ()
+  | opt :: ll when String.length opt > 0 && opt.[0] = '-' ->
+     coqdep_warning "unknown option %s" opt; parse ll
```

Comment:
The ones that are still used should not emit a warning, otherwise builds will be unreadable. I think only `-noinit` needs to be added to this category?

By the way, it is not clear if indeed coqdep will support `-noinit` or some other form in the future, as we actually improve the library model; indeed if `coqdep` some day gains the ability to include the implicit dep on `Init/Prelude.vo` , then `-noinit` would make total sense.

Reply:
To be clear, I suggest you add an extra line ignoring `-noinit` for now.